sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/RecognizeCustomFormsLiveTests.cs

```diff
@@ -598,13 +598,9 @@ public async Task StartRecognizeCustomFormsFromUriThrowsForNonExistingContent(bo
 
             await using var trainedModel = await CreateDisposableTrainedModelAsync(useTrainingLabels);
 
-            var operation = await client.StartRecognizeCustomFormsFromUriAsync(trainedModel.ModelId, invalidUri);
-
-            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () => await operation.WaitForCompletionAsync());
+            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () => await client.StartRecognizeCustomFormsFromUriAsync(trainedModel.ModelId, invalidUri));
 
-            Assert.AreEqual("2003", ex.ErrorCode);
-            Assert.True(operation.HasCompleted);
-            Assert.False(operation.HasValue);
+            Assert.AreEqual("2001", ex.ErrorCode);
```

Comment:
just wondering, did you see this change in both canary and prod?

Reply:
I saw it in both. Because I only run canary on the weekends, prod started failing on Friday, so idk if in the middle of the week Canary already had the changes